### PR TITLE
Update Spark and Accumulo

### PR DESCRIPTION
**Pull request by pomadchin**
Update up to Spark 1.5.2 and to Accumulo 1.7.0

### CODE REVIEW

**Comment by pomadchin** [comment]
Want to test it on a real Accumulo before merge.

**Comment by pomadchin** [comment]
Tested [here](https://github.com/pomadchin/accumulo-spark/tree/single-node-512m-accumulo-1.7) on a chatta demo project building it in a [java-8-container](https://hub.docker.com/r/daunnc/java-8-oracle/), will publish the complete guide how I use docker environment.

**Comment by lossyrob** [comment]
Doesn't 1.7 Accumulo have what we needed BatchInputFormat for? @echeipesh could we get rid of this?

**Comment by echeipesh** [comment]
Yep, we should be able to use this: https://accumulo.apache.org/1.7/apidocs/org/apache/accumulo/core/client/mapreduce/InputFormatBase.html#setBatchScan(org.apache.hadoop.mapreduce.Job,%20boolean)

We haven't changed our methodology since then.

**Comment by lossyrob** [comment]
Marking this as WIP, since we are going to test using `setBatchScan` instead of `BatchInputFormat`

**Comment by pomadchin** [comment]
Tested on Accumulo 1.6.4 and Accumulo 1.7.0, 2 node cluster configuration. :+1:

File: spark/build.sbt [hunk 1]
```diff
@@ -7,7 +7,8 @@ libraryDependencies ++= Seq(
     exclude("org.apache.hadoop", "hadoop-client"),
   "org.apache.spark" %% "spark-core" % Version.spark % "provided",
   "org.apache.hadoop" % "hadoop-client" % Version.hadoop % "provided",
-  "de.javakaffee" % "kryo-serializers" % "0.27",
+  "de.javakaffee" % "kryo-serializers" % "0.27" exclude("com.esotericsoftware.kryo", "kryo"),
```
Comment (pomadchin) [inline spark/build.sbt]: Can downgrade to `javakaffee 0.24` only after we decide what to do with `org.joda.time.Interval` (is it really necessary to use it)

**Pull request merged**